server/sonar-server/src/main/java/org/sonar/server/issue/notification/IssueChangeNotification.java

```diff
@@ -54,11 +54,12 @@ public IssueChangeNotification setIssue(DefaultIssue issue) {
   }
 
   public IssueChangeNotification setProject(ComponentDto project) {
-    return setProject(project.getKey(), project.longName(), project.getBranch());
+    return setProject(project.getKey(), project.longName(), project.getBranch(), project.getRootUuid());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getRootUuid"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)